homework4/task.py

```diff
@@ -0,0 +1,9 @@
+while True:
```

Comment:
Why do you use an infinite loop here?

Reply:
Here you can use an infinite loop, but there is no point of don't that in this case. But still, it was not a mistake, I just wanted to know why did you decide to do this